.pre-commit-config.yaml

```diff
@@ -40,6 +40,7 @@ repos:
     rev: v2.5.1
     hooks:
       - id: prettier
+        stages: [commit]
```

Comment:
Added because prettier does not support `commit_msg`

https://github.com/pre-commit/mirrors-prettier/issues/5